src/Accounts/Authentication/Authentication.csproj

```diff
@@ -12,7 +12,24 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.IdentityModel.Clients.ActiveDirectory" Version="3.19.2" />
+    <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="2.2.0" />
```

Comment:
We need to replace the AdalLogger above with some kind of adapter to get logging messages out of MSAL.